source/supported-hardware/mtl-mcl.rst

```diff
@@ -0,0 +1,139 @@
+.. MTL-MCL-board:
+
+|MTL-MCL| Board
+---------------------------------
+
+The Intel McLaren Island Reference Design Board (|mtl-mcl|) is an x86 maker board based on Intel platform Meteor Lake. The boards are used in IoT, industrial automation, digital signage areas, etc.
+
+Prerequisites
+^^^^^^^^^^^^^^^^
+
+|SPN| supports |Intel McLaren Island Reference Design Board|. To start developing |SPN|, the following equipment, software and environments are required:
+
+* |Intel McLaren Island Reference Design Board|
+* DediProg SF600 programmer
+* Linux host or windows host (see :ref:`running-on-linux` or :ref:`running-on-windows` for details)
+* Internet access
+
+.. |Intel McLaren Island Reference Design Board| raw:: html
+
+   <a href="https://www.seavo.com/en/products/products-info_itemid_558.html" target="_blank">Intel McLaren Island Reference Design Board</a>
+
+.. |instructions| raw:: html
+
+   <a href="https://wiki.up-community.org/BIOS_chip_flashing_on_UP_Squared" target="_blank">instructions</a>
+
+Board Setup
+^^^^^^^^^^^^^^^^^
+
+.. image:: /images/mcl-setup.jpg
+   :width: 600
+   :alt: |MTL-MCL| Board Setup
+   :align: center
+
+
+Before You Start
+^^^^^^^^^^^^^^^^^
+
+.. warning:: As you plan to reprogram the SPI flash, it's a good idea to backup the pre-installed BIOS image first.
+
+
+Boot the board and enter BIOS setup menu to get familiar with the board features and settings.
+
+Debug UART
+^^^^^^^^^^^
+
+For |MTL| platforms, serial port connector location can be found from the above table for each supported target board.
```

Comment:
@randylintw pls add the table or description of the serial port location.

Reply:
ok, document the location.